website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -73,12 +73,50 @@ $ operator-sdk create api \
-- Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones
-- Ensure that all roles configured in the `/deploy/roles.yaml` will be applied in the new project in the file `config/rbac/role.yaml`
-- If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
+Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`.
+
+### Migrating your Watches
+Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones. In our example, it will look like:
+
+```yaml
+# Use the 'create api' subcommand to add watches to this file.
+- group: example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remove the [Maker][marker] `+kubebuilder:scaffold:watch`. It allows the tool to update the watches file when new API's are created. 
+
+### Checking the Permissions (RBAC)
+
+SDK generated the roles for you in the `config/rbac/role.yaml`. However, you might need to  
+update this file with the permissions which are in your previous project in `/deploy/roles.yaml`.
+
+And then, beware that:
+
+- By default, the projects are `cluster-scoped`. Check that your role is of the `ClusterRole` Kind. For further information check the [operator-scope][operator-scope] doc.  
+
+- The `apiGroups` for `monitoring.coreos.com` is no longer required. If you would like to export the Prometheus metrics check the next section `Exporting metrics`.
+
+### Migrating your Helm Charts
+
+If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
+
+### Configuring your Operator
+
+In case your project has customizations in the `deploy/operator.yaml` then, it needs to be port to 
+`config/manager/manager.yaml`. However, note that the following env vars are no longer supported. 
+
+- `OPERATOR_NAME`: Now, the name of the operator can be defined due the flag `--project-name` when the project is 
+created. By default, it will be the current directory name. Also, see that it is stored in the PROJECT file. 
+- `WATCH_NAMESPACE`: By ddefault, it will be configured as `cluster-scoped`. To change it, use the flag `-leader-election-namespace` when you start the operator. 
+- `POD_NAME`:  By default, it will be the projectName stored in the PROJECT file. To change it, use the flag `-leader-election-id` when you start the operator. 
+
+**Note** The Dockerfile image also changes and now it is `distroless` and still been `rootless`. You might need to port some customizations made in your old project Dockerfile as well.
```

Comment:
This is incorrect. The Helm operator base image [still uses `ubi8/ubi-minimal`](https://github.com/operator-framework/operator-sdk/blob/35c871fa0ddf0ac473b20f58ac03e8897d6e38cd/hack/image/helm/Dockerfile#L1)

```suggestion
```

Reply:
Oh it is true.